src/ray/gcs/redis_context.cc

```diff
@@ -133,6 +135,71 @@ RedisContext::~RedisContext() {
   }
 }
 
+void GetRedisShards(redisContext *context, std::vector<std::string> *addresses,
```

Comment:
static void GetRedisShards to avoid polluted global namespace.

Reply:
fixed